modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportEntitiesWithPartiallyUpdate.testcase

```diff
@@ -42,11 +42,14 @@ definition {
 		}
 
 		task ("And Given "Stop the import on error" unchecked with Only Add New Records and valid JSON file with blog posting entry with existing externalReferenceCode") {
+			var siteName = TestCase.getSiteName();
```

Comment:
:+1:

Reply:
Never mind, just remember there is no default scope inlist, I think it's ok for now.
`From PM: There is no "default", I guess the list is ordered alphabetically,`